js/blocks/ExtrasBlocks.js

```diff
@@ -534,6 +534,8 @@ function setupExtrasBlocks(activity) {
                         }
                     }
                 }
+            } else if (logo.inMatrix) {
```

Comment:
Why not move this else if above the previous one so you don't need the && !logo.inMatrix ?

Reply:
Yes, i will do it. 👍